packages/dashboard-v2/src/components/FileList/useFormattedFilesData.js

```diff
@@ -10,7 +10,7 @@ const parseFileName = (fileName) => {
 
 const formatItem = ({ size, name: rawFileName, uploadedOn, downloadedOn, ...rest }) => {
   const [name, type] = parseFileName(rawFileName);
-  const date = dayjs(uploadedOn || downloadedOn).format("MM/DD/YYYY; HH:MM");
+  const date = dayjs(uploadedOn || downloadedOn).format("MM/DD/YYYY HH:MM");
```

Comment:
Let's use some more universal and unambiguous date format for example
- `YYYY-MM-DD HH:MM` ie. 2018-08-16 08:02
- `MMM D, YYYY HH:MM` ie. Aug 16, 2018 08:02

Reply:
bf55ca8f